integration/spark2/src/main/scala/org/apache/spark/sql/CarbonCatalystOperators.scala

```diff
@@ -101,13 +101,15 @@ case class UpdateTable(
     table: UnresolvedRelation,
     columns: List[String],
     selectStmt: String,
+    alias: Option[String],
```

Comment:
provide a default value for optional parameter

Reply:
Done